functions.js

```diff
@@ -77,7 +77,7 @@ function updateListe(data) {
       let completedClass = ''
       let checkboxChecked = ''
 
-      if (data[i]['done'] == "true") {
+      if ((data[i]['done'] == "true") || (data[i]['done'] == true)) {
```

Comment:
seems a glitch with the returned json - it will check for the string and boolean return now